src/navsections/K8sResourceSectionBlueprint/ResourceKindSectionNameCounter.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useMemo} from 'react';
+
+import {SectionCustomComponentProps} from '@models/navigator';
+
+import {useAppSelector} from '@redux/hooks';
+
+import {countResourceErrors, countResourceWarnings} from '@utils/resources';
+
+import * as S from '../../components/molecules/SectionRenderer/styled';
+import * as WarningStyle from '../../components/organisms/NavigatorPane/WarningAndErrorsDisplay.styled';
+
+function ResourceKindSectionCounter({sectionInstance, onClick}: SectionCustomComponentProps) {
+  const {id, isSelected, itemIds} = sectionInstance;
+  const isCollapsed = useAppSelector(state => state.navigator.collapsedSectionIds.includes(id));
+  const resourceMap = useAppSelector(state => state.main.resourceMap);
+  const resources = useMemo(() => itemIds.map(itemId => resourceMap[itemId]), [itemIds, resourceMap]);
```

Comment:
I assume an invariant of this codebase is that all resources can be found in the resource map, otherwise this might give some `undefined` values which I have to filter.

Reply:
It would be safer to filter out undefined values